src/parser/core/modules/WellPrepared.tsx

```diff
@@ -0,0 +1,73 @@
+import {Trans} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+
+// Give a use time of 5m10s between each potion use
+const TIME_BETWEEN_POTIONS_IN_MS = 310000
```

Comment:
What's the rationale for this?

Reply:
To further question this,

Why 5:10? Pots are 4:30 cooldown right now and the ideal time to use them (outside of super fast kills are at the full 2 minute burst windows for most jobs to catch party buffs.

So if anything you would want 4:30 + 1:30 for 6 minutes. There's no point in using a pot on cooldown if you aren't doing your full burst unless you are going to miss a use from it.